### DISCUSSION TITLE
Assets pipeline | Gerador estático

### DISCUSSION THREAD

**Discussion by agenciawdc** [frontendbr/forum#437] [Perguntas e Respostas]
Estou projetando um site para ser desenvolvido, usando um gerador estático, sem backend. Tenho duas dúvidas, se puderem me ajudar, vou ser grato pelo resto da vida a vocês kkkk.

Então.
   1. harps ou jekyll?
   2. como vocês fazem um bom versionamento dos arquivos js e css? 
       (Ex.: application.css?v=123123122i1ind12ifn1ifn1)

Abraço rapaziada!

**Comment by brunoravanhani** [comment]
Cabe uma terceira opção? Já usei Jekyll, hoje gosto mais do [Hexo](https://github.com/hexojs/hexo) por ser feito em js mesmo, apesar ter alguns bugs nos templates.

**Comment by thulioph** [comment]
Hoje tive esse dilema de qual utilizar.. Já usei middleman e jekyll, mas vinha procurando algo mais JavaScript e menos ruby.

Comecei no Hexo e tô gostando, vou aprimorar mais pra frente, porém até agora tudo fluindo como esperado.

**Comment by leonardoelias** [comment]
Galera, tem o Hugo tbm, feito em Go Lang

[https://gohugo.io/](https://gohugo.io/)

**Comment by RatoX** [comment]
Vei o Jekyll é muito bom pra poder 100% do GithubPages porém se tu tá criando um site pesado tipo mais de 5000 postagens ele começa a das umas engasgadas, principalmente se tu usa plugins. 

O Hexo é foda demais o problema é que se tu precisar rodar em modo increment em um servidor com forever , upstart ou até mesmo o runit tu vai ter muita dor de cabeça.

**Comment by felipefialho** [comment]
Sou apaixonado pelo Harp. Normalmente tenho mais simpatia por coisas feitas em JavaScript, hehe 😹

**Comment by simaodeveloper** [comment]
Uso Gulp para automatizar tudo, então como Template Engine acostumei com o Nunjucks da Mozilla.

**Comment by agenciawdc** [comment]
Pulei para o Hexo kkkk Estou gostando bastante pelo fato dele ser em Javascript também, então facilita muito, mas ainda acho que o Jekyll absurdamente mais fácil kkkk

**Comment by rubenmarcus** [comment]
Gostei muito desse aqui: https://nuxtjs.org/ Porque é feito em Vue. Alguem mais já usou ele?

**Comment by rubenmarcus** [comment]
@LFeh  abri uma [issue ](https://github.com/frontendbr/forum/issues/451) a toa aqui nao tinha visto essa

**Comment by felipefialho** [comment]
Sem problemas, coloquei label na outra.
